_posts/2017/2017-02-20-continuous-documentation-with-ansible.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title: "Continuous Documentation with Ansible"
```

Comment:
As this post has around 17800 characters, I prefer to split it into two posts.
For posts we recommend around 5000 characters.